src/main/cliapp/src/components/Actions/DuplicationAction.js

```diff
@@ -1,4 +1,4 @@
 import { Button } from "primereact/button";
 export const DuplicationAction = ({ props, handleDuplication, disabled }) => {
-  return <Button icon="pi pi-copy" disabled={disabled} className="p-button-text" onClick={() => handleDuplication(props)}/> ;
-}
\ No newline at end of file
+  return <Button icon="pi pi-copy" disabled={disabled} style={{padding:'0rem', fontSize: '1rem'}} className="p-button-text" onClick={() => handleDuplication(props)}/> ;
```

Comment:
Instead of adding style properties and doing the CSS inline, use the primeflex classes and add them to the className property.  For example, the style you have here could be add to the className as "p-0 text-base".

Reply:
Sure Adam